Source/Widgets/Cesium3DTilesInspector/Cesium3DTilesInspector.js

```diff
@@ -80,6 +80,21 @@ define([
         displayPanelContents.appendChild(makeCheckbox('showContentBoundingVolumes', 'Content Volumes'));
         displayPanelContents.appendChild(makeCheckbox('showRequestVolumes', 'Request Volumes'));
 
+        displayPanelContents.appendChild(makeCheckbox('pointCloudShading', 'Point Cloud Shading'));
+        var pointCloudShadingContainer = document.createElement('div');
+        pointCloudShadingContainer.setAttribute('data-bind', 'css: {"cesium-cesiumInspector-show" : pointCloudShading, "cesium-cesiumInspector-hide" : !pointCloudShading}');
+        pointCloudShadingContainer.appendChild(makeRangeInput('geometricErrorScale', 0, 2, 0.01, 'Geometric Error Scale'));
+        pointCloudShadingContainer.appendChild(makeRangeInput('maximumAttenuation', 0, 32, 1, 'Maximum Attenuation'));
+        pointCloudShadingContainer.appendChild(makeRangeInput('baseResolution', 0, 1, 0.01, 'Base Resolution'));
+        pointCloudShadingContainer.appendChild(makeCheckbox('eyeDomeLighting', 'Eye Dome Lighting (EDL)'));
```

Comment:
Could we enable the Eye Dome Lighting checkbox by default? In the usual case I like to turn on attenuation and EDL at the same time.

Reply:
Now enabled in `PointCloudShading.js`

Reply:
incentive to handle those translucent ones...